go.mod

```diff
@@ -1,16 +1,15 @@
 module github.com/crossplane/provider-template
 
-go 1.16
+go 1.17
 
 require (
-	github.com/crossplane/crossplane-runtime v0.13.0
-	github.com/crossplane/crossplane-tools v0.0.0-20201201125637-9ddc70edfd0d
-	github.com/google/go-cmp v0.5.2
+	github.com/crossplane/crossplane-runtime v0.14.1-0.20210812020058-ba474e81c62c
```

Comment:
@muvaf might be nice to use a pinned version for the template. Do we need to run a `crossplane-runtime` release to include some functionality this is bringing in?

Reply:
There isn't a specific thing that I need but it's good to keep it up-to-date. I can revert it to v0.14 but also run a release if you'd like to.